pkg/controller/controller_binding.go

```diff
@@ -58,11 +63,17 @@ func (c *controller) reconcileServiceBindingKey(key string) error {
 	}
 	binding, err := c.bindingLister.ServiceBindings(namespace).Get(name)
 	if apierrors.IsNotFound(err) {
-		glog.Infof("Not doing work for ServiceBinding %v because it has been deleted", key)
+		glog.Infof(
+			`%s "%s/%s": Not doing work for ServiceBinding %v because it has been deleted`,
```

Comment:
Message should be "...: not doing work because the ServiceBinding has been deleted"